server/pbench/bin/pbench-sync-satellite

```diff
@@ -110,6 +106,11 @@ mkdir -p $tmp
 mkdir -p $unpack
 
 log_init $(basename $0)
+if [[ -z "$mail_recipients" ]] ;then
+    echo "$PROG: mail_recipients is not defined" >&4
```

Comment:
For consistency's sake, this should have a $TS as well, but I'm not holding up the PR for this: we'll do it at the next opportunity,